src/Sylius/Behat/Page/Admin/Product/Common/ProductAssociationsTrait.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Page\Admin\Product\Common;
+
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Product\Model\ProductAssociationTypeInterface;
+
+trait ProductAssociationsTrait
```

Comment:
I would be for using FormElement instead of the trait

Reply:
Frankly I'd be for having a single `FormElement` instead of a trait per section + additional differences between simple/configurable. It'd be a bit of a bucket, but it could simplify the code in pages and contexts by a mile. We could probably even remove the UpdatePage split since the URI is the same for both of them.